src/Toolkit/A.php

```diff
@@ -31,6 +32,30 @@ public static function append(array $array, array $append): array
         return $array + $append;
     }
 
+    /**
+     * Loops through the array and resolves
+     * any item defined as Closure, applying
+     * the passed parameters
+     *
+     * @param array $array
+     * @param mixed $params parameters to pass to the Closure
+     * @return array
+     */
+    public static function apply(array $array, $params = []): array
+    {
+        if (is_array($params) === false) {
+            $params = A::wrap($params);
+        }
+
```

Comment:
I think the new PHP splat syntax makes this method much easier to use:

```suggestion
     * @param mixed ...$params Parameters to pass to the closures
     * @return array
     */
    public static function apply(array $array, ...$params = []): array
    {
```